bundles/org.openhab.ui.basic/src/main/resources/snippets/input.html

```diff
@@ -0,0 +1,25 @@
+<div class="mdl-form__row mdl-cell mdl-cell--6-col mdl-cell--8-col-tablet %visibility_class%">
+	<span class="mdl-form__icon">
+		<img data-icon="%category%" src="../icon/%category%?state=%state%&format=%icon_type%&anyFormat=true" />
+	</span>
+	<span %labelstyle% class="mdl-form__label">
+		%label%
+	</span>
+	<div
+		class="input__control mdl-form__control mdl-textfield mdl-js-textfield"
+		data-control-type="input"
+		data-item="%item%"
+		data-widget-id="%widget_id%"
+		for="oh-input-%item%"
+	>
+		<input
+			%valuestyle% type="text"
+			id="oh-input-%item%"
+			class="input__value input mdl-textfield__input"
+			value="%data_state%"
+		/>
+		<label for="oh-input-%item%" class="input__label input mdl-textfield__label">
+			%empty_state%
+		</label>
```

Comment:
I am not sure to understand what you want to display there ?
This is a part that apparently is not dynamically updated when receiving state updates.

Reply:
If there is no value yet, this label will be displayed as a placeholder. As soon as something is in the item state, or something is entered, it disappears. See mdl light text input widget.

Reply:
Ok, I will.
And I will test with a NULL/UNDEF state.

Reply:
Color and position look no perfect in that case:
![image](https://user-images.githubusercontent.com/10186704/221371962-1cc18b79-6a5d-47fe-8a00-eb51c1de29b3.png)

Reply:
You probably have to add `%valuestyle%` also on the label tag (input).

Reply:
I don’t think I should, because label can only represent an empty value and is grayed by default.

Reply:
Look at the column at left, the value in that case is not grayed.

Reply:
Sorry, didn’t see screenshots. Will check again.

Reply:
Could it have something to do with the dark theme?

Reply:
No this is the same with the default theme (white theme).

Reply:
It was due to the dark theme. I provided a fix.